ocrd/ocrd/cli/zip.py

```diff
@@ -39,13 +39,12 @@ def zip_cli():
               help='Basename of the METS file.',
               show_default=True)
 @click.option('-i', '--identifier', '--id', help="Ocrd-Identifier", required=True)
-@click.option('-I', '--in-place', help="Replace workspace with bag (like bagit.py does)", is_flag=True)
```

Comment:
This is a backwards-incompatible change. AFAICT the two aspects of the PR - dropping `--in-place` vs `ocrd zip update` - seem independent, so maybe we can have a separate PR for dropping `--in-place`, so we can warn potential users like @tboenig.

Reply:
Dropping `--in-place` is backwards incompatible indeed, but it does **not break** anything that has not **already**  been broken – is what Jonas is arguing IMHO correctly here, see https://github.com/OCR-D/core/issues/363.

Reply:
Since it isn't working, nobody will be inconvenienced by removing it, cannot argue with that logic, so LGTM.

Reply:
PR for dropping `--in-place` follows.